src/Doctrine/Annotation/Token.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Doctrine\Annotation;
+
+/**
+ * A Doctrine annotation token.
+ */
+class Token
+{
+    const T_NONE = 1;
```

Comment:
`T_NONE` is for whitespaces and meaningless words in the PHP comment, e.g. in
```php
/**
 * Some words
 */
```
`T_NONE` would apply to _Some_ and _words_